tests/FSharp.Compiler.ComponentTests/Miscellaneous/FsharpSuiteMigrated.fs

```diff
@@ -31,16 +31,17 @@ module ScriptRunner =
         let cu  = cu |> withDefines defaultDefines
         match cu with 
         | FS fsSource ->
-            File.Delete("test.ok")
+            // File.Delete("test.ok")
```

Comment:
Yeah so I've looked into that and I think this is just an old technique to verify that the some command executed without a problem. Modern test framework just explode automatically if this happens.

I might help you out here a bit, removing this at least from tests/fsharp :)

Reply:
It'd be great!